src/applications/disability-benefits/2346/containers/OrderTables.jsx

```diff
@@ -0,0 +1,233 @@
+                    <th className="form2346 vads-c-table__th">Model</th>
+                    <th className="form2346 vads-c-table__th">Serial Number</th>
+                    <th className="form2346 vads-c-table__th">Order Method</th>
+                    <th className="form2346 vads-c-table__th--narrow vads-u-text-align--center">
+                      Check to Order Socks
+                    </th>
+                  </tr>
+                  {dlcUISocksData.map(dlcUIObj => (
+                    <tr key={dlcUIObj.id}>
+                      <td>{dlcUIObj.dateProcessed}</td>
+                      <td>{dlcUIObj.productGroup}</td>
+                      <td>{dlcUIObj.product}</td>
+                      <td>{dlcUIObj.id}</td>
+                      <td>{dlcUIObj.orderIntakeMethod}</td>
+                      <td>{dlcUIObj.checkbox} </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table> */
+              }
+              <table id="accessoriesTable" className="form2346 vads-c-table">
+                <tbody>
+                  <tr>
+                    <th className="form2346 vads-c-table__th">
+                      Date Processed
+                    </th>
+                    <th className="form2346 vads-c-table__th">Product Type</th>
+                    <th className="form2346 vads-c-table__th">Model</th>
+                    <th className="form2346 vads-c-table__th">Serial Number</th>
+                    <th className="form2346 vads-c-table__th">Order Method</th>
+                    <th className="form2346 vads-c-table__th--narrow vads-u-text-align--center">
+                      Check to Order Accessories
+                    </th>
+                  </tr>
+                  {dlcUIAccessoriesData.map(dlcUIObj => (
+                    <tr key={dlcUIObj.id}>
+                      <td>{dlcUIObj.dateProcessed}</td>
+                      <td>{dlcUIObj.productGroup}</td>
+                      <td>{dlcUIObj.product}</td>
+                      <td>{dlcUIObj.id}</td>
+                      <td>{dlcUIObj.orderIntakeMethod}</td>
+                      <td>{dlcUIObj.checkbox} </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+              <Link to="/comments">
+                <button
+                  type="button"
+                  className="usa-button-primary va-button-primary"
+                >
+                  Back
+                </button>
+              </Link>
+              <Link to="/home">
+                <button
+                  type="button"
+                  className="usa-button-primary va-button-primary"
+                >
+                  Submit
+                </button>
+              </Link>
+            </>
+          )}
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => ({
+  dlcBatteryData: state?.form2346Reducer?.dlcBatteryData || '',
```

Comment:
It's safe to assume that `state` will exist and it's a slightly better practice to use the nullish coalescing operator instead of logical or (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). So I'd rewrite this as `state.form2346Reducer?.dlcBatteryData ?? ''`